app/Events/StandAssignedEvent.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Events;
+
+use App\Models\Stand\StandAssignment;
+use Illuminate\Broadcasting\Channel;
+use Illuminate\Broadcasting\PrivateChannel;
+use Illuminate\Contracts\Broadcasting\ShouldBroadcast;
+
+class StandAssignedEvent implements ShouldBroadcast
+{
+    const CHANNEL = 'stand-assignments';
+
+    /**
+     * @var StandAssignment
+     */
+    private $standAssignment;
```

Comment:
Plz can we 7.4 soon?

Reply:
We're running GH Actions against it but alas... not yet :(